lms/static/js/discussions_management/models/course_discussions_settings.js

```diff
@@ -0,0 +1,14 @@
+(function(define) {
```

Comment:
I don't understand models/divide_discussions.js and how it relates to this model. Can you explain?